syntax = "proto3";
package emails;
service Emails {
  rpc SendMail (SendReq) returns (SendRes) {}
}
message SendReq {
  string email = 1;
  string to = 2;
  string subject = 3;
  string body = 4;

}
message SendRes {
  int32 status = 3;
  string message = 4;
}
